Testreglar/253/Nett/nett-2.5.3a.json

```diff
@@ -0,0 +1,153 @@
+		},
+		{
+			"title": "3.2"
+		},
+		{
+			"title": "3.3"
+		}
+	],
+	"side": "2.1",
+	"element": "3.1",
+	"steg": [
+		{
+			"stegnr": "2.1",
+			"spm": "Hvilken side tester du?",
+			"ht": "<p>Angi URL eller side-ID.</p>",
+			"type": "tekst",
+			"kilde": [],
+			"label": "URL/Side-ID:",
+			"datalist": "Sideutvalg",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			}
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Har testsiden interaktive brukergrensesnittkomponenter? ",
+			"ht": "<p>Gjøre visuell inspeksjon og identifiser interaktive brukergrensesnittkomponenter. Dette gjelder blant annet brukergrensesnittkomponenter med rolle <a href=\"https://www.w3.org/TR/wai-aria-1.1/#widget_roles\" target=\"_blank\" rel=\"noopener\">widget</a>.</p>\n<p><strong>Merk: </strong>Skjema i PDF, Word eller liknende, er ikke aktuelle testobjekt.</p>",
+			"type": "jaNei",
+			"kilde": [],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "2.3"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Testsiden har ikke interaktive brukergrensesnittkomponenter."
+				}
+			}
+		},
+		{
+			"stegnr": "2.3",
+			"spm": "Hvilken type brukergrensesnittkomponenten er det?",
+			"ht": "<p>Velg fra alternativ:</p>",
+			"type": "radio",
+			"kilde": [],
+			"svarArray": [
+				"Skjemaelement",
+				"Meny",
+				"Lenke",
+				"Knapp",
+				"Andre komponenter"
+			],
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.4"
+				}
+			}
+		},
+		{
+			"stegnr": "2.4",
+			"spm": "Er det synlig ledetekst, rett ved siden av brukergrensesnittkomponenten, som består av tekst eller bilde av tekst?",
+			"ht": "<p>Synlig ledetekst til brukergrensesnittkomponenten kan bestå av</p>\n<ul>\n<li>bare tekst</li>\n<li>bilde av tekst brukt som symbol, for eksempel\n<ul>\n<li>Der bokstavene <strong>F</strong>, <em>K</em> og <span style=\"text-decoration: underline;\">U</span> brukes i et tekstredigeringsverktøy som symbol for fet, kursiv og understreket tekst</li>\n<li>Forstørrelsesglass symbol for søkefelt</li>\n<li>Hjertesymbol for funksjonen favoritt</li>\n</ul>\n</li>\n</ul>\n<p>Rett ved siden av brukergrensesnittkomponenten betyr vanligvis ledetekst som står</p>\n<ul>\n<li>rett til vestre for kombinasjonsbokser, nedtrekkslister og tekstfelt</li>\n<li>rett til høyre for avkryssingsbokser og radioknapper</li>\n<li>i lenker, faner. knapper eller rett under ikon eller bilde av tekst som fungerer som knapper</li>\n<li>placeholdertekst i tekstfelt - <strong>Merk:</strong> Dette gjelder kun i relasjon til 2.5.3, når synlig ledetekst mangler.</li>\n</ul>\n<p><strong>Merk:</strong> Dette skal du ikke ta med i vurderingen av synlig ledetekst i relasjon til krav 2.5.3</p>\n<ul>\n<li>gruppeledetekst for skjemaelementer i grupper\n<ul>\n<li>dvs. ledetekst brukt med legend/fieldset eller med rolle som gruppe eller radiogruppe. </li>\n</ul>\n</li>\n<li>instruksjoner</li>\n<li>overskrifter</li>\n</ul>",
```

Comment:
Jeg kan se ud fra Understanding-dokumentet hvorfor "tekst brugt som symbol" er inkluderet i denne test-regel, da Understanding 2.5.3 beskriver ret udførligt, hvilke tilgængelighedskrav "tekst brugt som symbol" skal leve op til.
Jeg vil jeg dog mene, at "tekst brugt som symbol IKKE er en del af scope for 2.5.3 ud fra denne indledning til afsnittet i Understanding 2.5.3: 

"For the purposes of this SC, text should not be considered a visible label if it is used in a symbolic manner, rather than directly expressing something in human language as per the definition of [text](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html#dfn-text) in WCAG."


I ACT-reglen de-scopes denne type indhold i Expectation: "except for characters in the [text nodes](https://dom.spec.whatwg.org/#text) used to express [non-text content](https://www.w3.org/TR/WCAG21/#dfn-non-text-content)". 
Dette skyldes, at Applicability dermed kunne gøres fuldautomatisk, så manuelle vurderinger begrænses til Expectation.

Reply:
Basert på samme setning går "tekst brukt som symbol" i applicability, men selve tekst i slik situasjon skal ikke antas som "synlig ledetekst" som videre skal sjekkes i tilgjengelig navn. 
I et slikt tilfelle skal det tilgjengelige navnet være funksjonen knappen tjener, ikke de synlige symbolske tekst tegnene/bokstavene. Og det er det som vi tester her i denne testregelen (videre i steg 3.4).